packages/teleport/src/stores/storeUserContext.ts

```diff
@@ -122,6 +122,10 @@ export default class StoreUserContext extends Store<UserContext> {
     return this.state.accessRequestId;
   }
 
+  getAccessRequests() {
```

Comment:
to follow naming convention `getAccessRequestAccess` :laughing: (serious i'm okay with it) or `getAccessRequestPerms` (otherwise the current name implies we are getting a list of requests)